_data/l10n.yml

```diff
@@ -77,7 +77,9 @@ it:
 
     home_tools_title: Strumenti
     home_tools_subtitle: Ci appoggiamo a strumenti già esistenti e molto conosciuti per fornirti le informazioni, il codice e il supporto di cui hai bisogno
-
+    
+    postarticle_socials_icon_title_prefix: Condividi su
```

Comment:
maybe there is no need to create this, we could do as we did here

https://github.com/italia/developers.italia.it/blob/master/_includes/share.html#L3

Reply:
@sebbalex Nice catch!

Reply:
@sebbalex I guess something like
`{{ t.share_msg | capitalize }} {{ t.on_ }} {{ social[0] | capitalize }}`
should render to `Share on Twitter`.
I'll make the changes.